### ISSUE TITLE
Remove the information about meetings from the readme

### ISSUE THREAD

**Issue by ozkriff** [rust-gamedev/wg#95]
It seems like no one participates in these meetings for many months now, right?

**Comment by Wodann** [comment]
Yes, that's correct. There haven't been any meetings about WG topics for quite some time now.

**Comment by Wodann** [comment]
There has been sporadic traffic on matrix, but even less than on our Discord channel.

I don't see any objection against removing it
